contrib_bots/lib/john.py

```diff
@@ -0,0 +1,118 @@
+# pip install chatterbot is necessary
+import json
+import os
+import time
+
+from random import choice
+
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer, ListTrainer
+
+JOKES_PATH = os.path.expanduser('lib/John/jokes.json')
+DATABASE_PATH = os.path.expanduser('~/JohnBot/database.db')
```

Comment:
Hi @femnad . Did you changed the code when running the bot? I tried running it with your command and it worked fine, however when I tried changing the directory to ./JohnBot, when the database was created it was empty and thus, ended on an infinite loop. It seems it have issues finding hidden directories, however it can work with hidden files on the directory from where it's running. If you would prefer that I can make the changes but I'm a little behind with tasks. May mentors please accept this task so that I can continue with others?